docs/moderator-handbook.md

```diff
@@ -249,122 +249,122 @@ Here's how moderators deal with violations of our [Code of Conduct](https://code
+   For example:
 
    - Camper posts a wall of code to request help:
 
-     Moderator: @username Please use CodePen or Pastebin when posting large amounts of code.
+     Moderator: **@username** Please use CodePen or Pastebin when posting large amounts of code.
 
    - Or if you really have to explain why:
 
-     Moderator: @username Please use CodePen or Pastebin when posting large amounts of code, because it disrupts the chat for everyone and could be considered spamming according to our Code of Conduct.
+     Moderator: **@username** Please use CodePen or Pastebin when posting large amounts of code, because it disrupts the chat for everyone and could be considered spamming according to our [Code of Conduct](https://code-of-conduct.freecodecamp.org).
 
-   - For mild and unintentional violations of the code of conduct:
+   - For mild and unintentional violations of the [Code of Conduct](https://code-of-conduct.freecodecamp.org):
 
-     Moderator: This is a friendly reminder for everyone to follow the code of conduct: https://code-of-conduct.freecodecamp.org/
+     Moderator: This is a friendly reminder for everyone to follow the [Code of Conduct](https://code-of-conduct.freecodecamp.org): https://code-of-conduct.freecodecamp.org/
 
 7. **Don’t brag about being a moderator**
 
-   Do not see yourself as above the community. You are the community. And the community has trusted you to help protect something rare that we all share - a _welcoming_ place for new developers.
+   Do not see yourself as above the community. **You are the community.** And the community has trusted you to help protect something rare that we all share - a _welcoming_ place for new developers.
 
    If you brag about being a moderator, people may feel uneasy around you, in the same way that people may feel uneasy around a police officer, even if they’re doing nothing wrong. This is just human nature.
 
 8. **Don’t contradict other moderators**
 
-   If you disagree with a moderator's action, talk with them in private or bring it up in the #mod-chat channel. Never override a moderator's action, and never contradict the other moderator(s) publicly. Instead, have a cool-headed discussion in `#mod-chat` and convince the moderator that they themselves should reverse their ban or change their point of view.
+   If you disagree with a moderator's action, talk with them in private or bring it up in the #mod-chat channel. Never override a moderator's action, and never contradict the other moderator(s) publicly. Instead, have a cool-headed discussion in `#mod-chat` and convince the moderator that they themselves should reverse their ban or change their PoV (Point of View).
 
-   Remember: we’re all on the same team. We want to dignify the role of moderators and present a unified front.
+   *Remember: We’re all on the same team. We want to dignify the role of moderators and present a unified front.*
```

Comment:
```suggestion
   _Remember: We’re all on the same team. We want to dignify the role of moderators and present a unified front._
```

Think this was my fault. Prettier expects underscores for italics instead of asterisks. 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix